src/main/java/EventsAndCommands/ChatEventHandler.java

```diff
@@ -0,0 +1,47 @@
+package EventsAndCommands;
+
+
+import net.dv8tion.jda.core.events.message.guild.GuildMessageReceivedEvent;
+
+
+/**
+ * This class provides the base structure of a chatevent that might be handled. 
+ * To implement a new ChatEventHandler it is sufficient to just inherit from this 
+ * Class and override {@see ChatEventHandler#trigger(GuildMessageReceivedEvent)}
+ * and {@see ChatEventHandler#receiveCommand(GuildMessageReceivedEvent)}.
+ */
+public abstract class ChatEventHandler {
+
+    public ChatEventHandler(){
+        register();
+    }
+    /**
+     * This method is executed when the command got triggered trough {@see ChatEventHandler#trigger(GuildMessageReceivedEvent)}.
+     * This method should only handle what happens after the command was triggered, i.e. responses or whatever,
+     * @param event the message that triggered the event.
+     */
+    public abstract void receiveCommand(GuildMessageReceivedEvent event);
```

Comment:
Does this method execute a command If so `execute` might be nicer. If not `onCommand` might fit better? Not sure.